src/routes/uniswap.route.js

```diff
@@ -129,6 +131,7 @@ router.post('/buy-price', async (req, res) => {
       })
     }
   } catch (err) {
+    console.log(err)
```

Comment:
Since we're using `debug` for other print statements in Gateway, we should also use `debug` here